src/frontend/delete-github-subscription.ts

```diff
@@ -71,14 +75,24 @@ export default async (req: Request, res: Response): Promise<void> => {
 
       await subscription.destroy();
       await submitProto(action);
+
+      tags.push(`status:202`)
+      statsd.increment('delete-github-subscription.success', tags);
```

Comment:
We can potentially extract it as a middleware function and apply it to all our endpoints. 

Also, many of these metrics are coming out of the box from load balancers and service proxy. I believe we can just use existing platform metrics to count success/failures. 

For example: some client metrics emitted by Service Proxy: https://atlassian.signalfx.com/#/chart/v2/EwuFjPTA0AE?groupId=EUe6Xq7A4AA&configId=EUe6Kd_A0AA&startTime=-12h&endTime=Now

Load Balancer metrics: https://atlassian.signalfx.com/#/chart/v2/EUe5LRSAwAA?groupId=EUe6Xq7A4AA&configId=EUe6GqPA0AA&startTime=-12h&endTime=Now